lib/config/get.js

```diff
@@ -1,10 +1,20 @@
 const path = require('path')
-const { getInstalledPathSync } = require('get-installed-path')
+const { curry, flow, get, partialRight } = require('lodash/fp')
```

Comment:
Curry is not used - or is this your signature style to always have it someplace? ;-)

Reply:
Dang, I really need to get some eslint going.

Reply:
I do enjoy applying curry everything however.